webapp/components/survey/CategorySelector/CategorySelector.js

```diff
@@ -4,20 +4,22 @@ import React, { useState, useEffect, useCallback } from 'react'
 import PropTypes from 'prop-types'
 
 import * as A from '@core/arena'
+import * as StringUtils from '@core/stringUtils'
 import * as Category from '@core/survey/category'
 
 import { useI18n } from '@webapp/store/system'
 import { useSurveyId } from '@webapp/store/survey'
 import * as API from '@webapp/service/api'
 import { TestId } from '@webapp/utils/testId'
 
+import { useNotifyWarning } from '@webapp/components/hooks'
 import Dropdown from '@webapp/components/form/Dropdown'
 import PanelRight from '@webapp/components/PanelRight'
 import CategoryList from '@webapp/components/survey/CategoryList'
 import CategoryDetails from '@webapp/components/survey/CategoryDetails'
 import ButtonMetaItemAdd, { metaItemTypes } from '@webapp/components/survey/ButtonMetaItemAdd'
 
-const CategorySelector = (props) => {
+export const CategorySelector = (props) => {
```

Comment:
this is just a trying to avoid the eslint func-names rule to pass, but it doesn't work too...